dotcom-rendering/src/web/components/CardHeadline.tsx

```diff
@@ -252,6 +282,11 @@ export const CardHeadline = ({
 									? 'bold'
 									: 'regular',
 						  }),
+					format.theme !== ArticleSpecial.Labs &&
+						fontStylesOnMobile({
+							size: sizeOnMobile || size,
+							fontWeight: containerPalette ? 'bold' : 'regular',
+						}),
```

Comment:
I could be confused here (very possible) but is it that we only want to set this if `sizeOnMobile` exists?

If so, could we guard against undefined on this before running the function so we don't need to fallback to `size`? A bit like:

```
						sizeOnMobile && fontStylesOnMobile({
							size: sizeOnMobile,
							fontWeight: containerPalette ? 'bold' : 'regular',
						}),
```

Reply:
I think we'd like to be able to fallback to size, it means that we don't need to change any of existing Components that use Card. In almost all cases we can just use the normal `size` for deciding what style should be applied on mobile, its only in some very few scenarios where we actually need to specify what `sizeOnMobile` should be.

Reply:
Ah, I think I see what's happening here now. We _always_ want to set _both_ `fontStyles` **and** `fontStylesOnMobile`. But sometimes we override the `size` value when setting `fontStylesOnMobile`.

This all makes sense now 👍